docs/use-cases/tokenization/index.page.tsx

```diff
@@ -0,0 +1,389 @@
+import React, { useRef, useState } from "react";
+import { useTranslate } from "@portal/hooks";
+import { NavList } from "shared/components/nav-list";
+
+const { translate } = useTranslate();
+
+export const frontmatter = {
+  seo: {
+    title: "XRPL Overview",
+    description:
+      "An introduction to the key benefits and features of the XRP Ledger, a public blockchain driven by the XRPL developer community.",
+  },
+};
+
+const useCases = [
+  {
+    description: "Stablecoin Issuer",
+    link: "/docs/use-cases/tokenization/stablecoin-issuer/",
+  },
+  {
+    description: "NFT Marketplace",
+    link: "/docs/use-cases/tokenization/nftoken-marketplace/",
+  },
+  {
+    description: "Authorized Minter",
+    link: "/docs/use-cases/tokenization/authorized-minter/",
+  },
+  {
+    description: "Digital Artist",
+    link: "/docs/use-cases/tokenization/digital-artist/",
+  },
+];
+
+const securityAdvantages = [
+  {
+    title: "Trustlines & Authorized Trustlines",
```

Comment:
It's strange to me that these cards aren't links nor do they contain links to read more about the features they tout.

For this one, I suggest linking to `/docs/concepts/tokens/fungible-tokens/`